applications/jonny5/src/j5_channels.erl

```diff
@@ -91,6 +96,12 @@
                      ,{{?MODULE, 'handle_rate_resp'}
                        ,[{<<"rate">>, <<"resp">>}]
                       }
+                     ,{{?MODULE, 'handle_channel_destroy'}
+                       ,[{<<"call_event">>, <<"CHANNEL_DESTROY">>}]
+                      }
+                     ,{{?MODULE, 'handle_channel_destroy'}
+                       ,[{<<"call_event">>, <<"CHANNEL_DISCONNECTED">>}]
```

Comment:
I think you can put this tuple in the list above this one to only have one tuple with two events routed to the callback.